Mathmagician/Program.cs

```diff
@@ -13,41 +13,109 @@ static void Main(string[] args)
 
             string prompt = "> ";
 
-            Console.WriteLine("Let's cook some math!");
-            Console.WriteLine("Your choices are integers, primes, evens, odds, or Fibonnaci numbers.");
-            Console.WriteLine("What type of number would you like me to display?");
-            Console.Write(prompt);
-            
-            // Take the user's input command and store it as a variable (userRequest)
-            string userRequest = Console.ReadLine();
-            Console.WriteLine(userRequest + System.Environment.NewLine);
 
-            Console.WriteLine("How many numbers shall I display?");
-            Console.Write(prompt);
 
-            // Take the user's input and store it as another variable (numberToWrite)
-            int numberToWrite = Int32.Parse(Console.ReadLine());
 
-            // Logic here (probably a switch statment) that takes the stored input and uses it to call a method
+            Console.WriteLine("LET'S COOK SOME MATH!");
 
+            bool goAgain = true;
 
-            // INTEGER METHOD
-            Integer my_int = new Integer();
-            my_int.WriteIntegersToConsole(my_int.Make_Integers(numberToWrite));
-            Console.ReadLine();
+            while (goAgain)
+            {
+                Console.WriteLine("Your choices are integer, prime, even, odd, or Fibonacci numbers.");
+                Console.WriteLine("What type of number would you like me to display?");
+                Console.Write(prompt);
 
-            // ODD METHOD
+                // Take the user's input command and store it as a variable (userRequest)
+                string showThisNumberType = null;
 
+                while (showThisNumberType == null)
```

Comment:
The loop checks the user's input for a valid request (i.e., did the user ask for one of the number types we're planning to provide) and responds accordingly.  If the user gave a valid request, an "Okay, we'll do X numbers" message is written to the console.  If the user gave an invalid request -- including a misspelled option -- the loop prompts for a new response.  Some user responses will result in exiting the program (see lines 46 - 50).